### ISSUE TITLE
Why keda scaler only supports scale applications based on Redis Lists and Redis Stream, not on Redis Key-Values?

### ISSUE THREAD

**Issue by excbreezeair** [kedacore/keda#5003]
### Proposal

Do you have a plan? Thank you.

### Is this a feature you are interested in implementing yourself?

No

**Comment by tomkerkhove** [comment]
Because nobody asked for it? :)

**Comment by tomkerkhove** [comment]
Are you willing to contribute it?

**Comment by vinod827** [comment]
@zroubalik @tomkerkhove I'm interested in working on this. Can I pick this up?

**Comment by JorTurFer** [comment]
yeah! thanks @vinod827

**Comment by Binternet** [comment]
+1 on this
@vinod827 any progress? can I help with something?

**Comment by vinod827** [comment]
@Binternet yes, I have gone through with it...and looks good to me...I think we should also add rate limiting and cooldown period to avoid the unnecessary scaling up/down
